custom_components/miele/const.py

```diff
@@ -509,7 +509,7 @@
     24555: "appliance_settings",  # total long coffee
     24556: "appliance_settings",  # total italian cappuccino
     24557: "appliance_settings",  # total latte macchiato
-    24558: "appliance_settings",  # total caffellatte
+    24558: "appliance_settings",  # total caffe_latte
```

Comment:
Based on the spelling of the other comments, this should be `caffe latte` without the underscore.